use_cases/LWRS/train/ARMA_train_Braidwood_2018_2021_periodic.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" ?>
```

Comment:
Would you mind adding the relevant information and links to the ARMA catalog we have here in FORCE?

Reply:
What do you mean by this? Could you give me an example of what should be done?

Reply:
There is a table cataloging all the ARMA models in FORCE. 

You can edit the file `MODELS_INDEX.MD` with the relevant information contained in the table (for each ARMA you've trained in this use case). 

https://github.com/idaholab/FORCE/blob/main/use_cases/MODELS_INDEX.md